cfme/services/catalogs/catalog_item.py

```diff
@@ -244,9 +244,8 @@ def create(self):
             provisioning_view.fill_with(self.provisioning_data)
 
         view.add_button.click()
-        view.flash.assert_success_message('Service Catalog Item "{}" was added'.format(self.name))
         view = self.create_view(AllCatalogItemView)
-        assert view.is_displayed
+        view.flash.assert_success_message('Service Catalog Item "{}" was added'.format(self.name))
```

Comment:
I think you should still assert that the view is displayed here. Moving the flash message check is perfect though.